scripts/apidoc/output/constants.ts

```diff
@@ -0,0 +1 @@
+export const scriptCommand = 'pnpm run generate:api-docs';
```

Comment:
I prefer to name such constants with `CONSTANT_CASE`

```suggestion
export const SCRIPT_COMMAND = 'pnpm run generate:api-docs';
```